hoomd/mpcd/SphereGeometry.h

```diff
@@ -22,7 +22,7 @@ namespace detail
 {
 //! Sphere geometry
 /*!
- * This models a fluid confined inside a sphere, centered at the origin and with radius R.
+ * This models a fluid confined inside a sphere(moving with velocity V), centered at the origin and has radius R at end of streaming step.
```

Comment:
I think in the derivation of collision we assumed V<0 (that means m_V here should be negative ) and the formula reduced to the formula we had earlier when V= 0 . Do you mean doing an error check if Geometry is expanding instead of Shrinking ?